bin/benchmark-bootstrap.sh

```diff
@@ -83,7 +83,7 @@ CP=${CP}":${SCRIPT_DIR}/../libs/*"
 #
 # ADD YOUR/CHANGE ADDITIONAL OPTIONS HERE
 #
-JVM_OPTS="-Xms2g -Xmx2g -server -Djava.net.preferIPv4Stack=true "${JVM_OPTS}
+JVM_OPTS="-Djava.net.preferIPv4Stack=true "${JVM_OPTS}
```

Comment:
Seems this not related to the issue?

Reply:
Yes, but this applies to overriding JVM options. They are configured in .properties. Their presence is not justified here.

Reply:
Feel free to propose additional PR with this fix.
Let reviewers check one fix per PR.